airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryDestination.java

```diff
@@ -190,16 +190,17 @@ public static BigQuery getBigQuery(final JsonNode config) {
   }
 
   public static GoogleCredentials getServiceAccountCredentials(final JsonNode config) throws IOException {
-    if (!BigQueryUtils.isUsingJsonCredentials(config)) {
+    final JsonNode serviceAccountKey = config.get(BigQueryConsts.CONFIG_CREDS);
+    if (serviceAccountKey == null) {
       LOGGER.info("No service account key json is provided. It is required if you are using Airbyte cloud.");
       LOGGER.info("Using the default service account credential from environment.");
```

Comment:
// Follows this order of resolution 
      // https://cloud.google.com/java/docs/reference/google-auth-library/latest/com.google.auth.oauth2.GoogleCredentials#com_google_auth_oauth2_GoogleCredentials_getApplicationDefault__

Tried to add to the file but couldn't figure how to add suggestion

Reply:
added comment via https://github.com/airbytehq/airbyte/pull/34073/commits/d83cdc4113d975c0deca54255dc560ced80867d6